test/org/traccar/protocol/GpsmtaProtocolDecoderTest.java

```diff
@@ -19,6 +19,15 @@ public void testDecode() throws Exception {
         verifyPosition(decoder, text(
                 "359144048138856 1442932957 49.85064 24.003979 1 0 40 0 10 110 26 0 0"));
 
+        verifyPosition(decoder, text(
+                "359144048138856 1442932957 -49.85064 24.003979 1 0 40 0 10 110 26 0 0"));
+
+        verifyPosition(decoder, text(
```

Comment:
I don't think we need tests for all possible combinations. Just one would be enough.